README.md

```diff
@@ -54,46 +54,58 @@ For displaying the quizzes themselves, we will be storing them in a json file an
 
 We are using [google firestore](https://firebase.google.com/) to store the data entered. A document store (nosql database) gives us the flexibility to store this quiz data however we see fit. We can adjust the model in the future with very little cost.
 
-Here is the initial data model for our quiz answer:
+Firestore breaks things down into `collections` and `documents`. You'll notice a version field in the object. This gives us a way to change the quiz and know exactly which quiz they took. We give each quiz a guid so we can identify them if we ever want to add another quiz in the future.
+
+We will have a top-level `users` collection. This collection will hold `user` documents. Each `user` document will look like this:
 
 ```js
 {
-  createdAt: '2021-05-28'
-  guid: '92db5f82-0390-40fc-b078-70e7594c8043'
-  version: 0
-  answers: [
-    {
-      guid: 'b6656926-f6af-4a95-b7aa-b695c53f610f'
-      question: 'How many days in a row have you trained?',
-      answer: '2 days'
-      score: 3
-    }
-  ]
+  users: [{
+    id: 'some-id',
+    currentAnswer: 'quiz-answer-guid-02',
+    answers: [
+      {id: 'quiz-answer-guid-01', totalScore: 75.0}
```

Comment:
Im not sure about adding the `totalScore` here but it seems like it would be useful because thats the most important piece of data for an `answer`.